themes/wporg-support/style-rtl.css

```diff
@@ -3634,7 +3635,7 @@ section {
 .bbpress #bbpress-forums .wporg-bbp-user-notes-toggle,
 .bbpress #bbpress-forums .wporg-bbp-user-flag {
 	font-size: 0.8rem;
-	font-weight: normal;
+	font-weight: 285;
```

Comment:
Why 285?

Reply:
Good question :thinking:  That was generated by grunt (the one located at the theme folder). Should I run a script located elsewhere?